UnityWeld/Binding/AbstractMemberBinding.cs

```diff
@@ -175,10 +175,10 @@ protected void ParseViewEndPointReference(string endPointReference, out string m
         /// Standard MonoBehaviour awake message, do not call this explicitly.
         /// Initialises the binding.
         /// </summary>
-        protected void Awake()
-        {
-            Init();
-        }
+        //protected void Awake()

```

Comment:
Should just delete this. We try to avoid leaving blocks of commented-out code because it can be confusing to someone else reading the code.